store/store.go

```diff
@@ -51,12 +55,22 @@ const (
 
 // BoltStore represents the access type for store
 type BoltStore struct {
-	db *bolt.DB
+	db            *bolt.DB
+	casbinAdapter persist.BatchAdapter
+}
+
+// SettingsStore is the interface that defines methods needed to save settings config into the store.
+type SettingsStore interface {
+	SettingsPut(config *gaia.StoreConfig) error
```

Comment:
why can't an RBACPut be a SettingsPut? Why are they two different functions? An interface shouldn't be explicit. It should be generic mostly. If there is a settingsRbacPut that should most likely be either a more specialised interface or a concrete implementation of settings put.

I see that it's putting / getting an RBACConfig instead of a StoreConfig, but that also means that it can't be a SettingsStore. 

What I mean is, either make rbac use settingsput or have a separate interface. :)